content/applications/finance/fiscal_localizations/brazil.rst

```diff
@@ -0,0 +1,371 @@
+--> Configuration --> Settings --> Taxes` and add the email address you want to use to log in to the
+Avatax portal, and the click on :guilabel:`Create a new account`. This email will be used as the
+administrator email address in Avatax.
+
+After you create the account from Odoo, you need to go to the Avalara Portal to set up your
+password.
+
+.. warning::
+   Only create an account with the Avatax portal email you want to use in production when you are in
+   the database that will have your production transactions (real transactions). If you want to
+   test, you can use a dummy email (not a real one) to create the account. This will bring testing
+   credentials for you to test on a testing or sandbox database.
+
+.. tip::
+   If you do not know how to log in for the first time into the Avatax portal, `click here
+   <https://drive.google.com/file/d/1jPtKzilfLrfdRXyPGacQNr6i2zvQvoOd/view?usp=sharing>`_. If you
+   want to start using Avatax in Odoo without creating a password and accessing the Avalara Portal,
+   you will be able to. From Odoo the only step needed is creating an account to start using the
+   Avalara Tax Computation Engine.
+
+.. image:: brazil/avatax-account-configuration.png
+   :align: center
+   :alt: Avatax account configuration.
+
+.. note::
+   You can transfer API credentials. Use this only when you already created an account in another
+   Odoo instance and wish to reuse it.
+
+Configure master data
+---------------------
+
+Chart of accounts
+~~~~~~~~~~~~~~~~~
+
+The :doc:`chart of accounts </applications/finance/accounting/get_started/chart_of_accounts>` is
+installed by default as part of the set of data included in the localization module, the accounts
+are mapped automatically in taxes, default account payable, and default account receivable.
+
+The chart of accounts for Brazil is based on the SPED CoA, this gives a baseline of the accounts
+needed in Brazil.
+
+You can add or delete accounts according to the company's needs.
+
+Taxes
+~~~~~
+
+Taxes will be automatically created when installing the Brazilian localization. Taxes will be
+already configured, and some of them will be used by Avalara when computing taxes on the sale order
+or invoice.
+
+Taxes can be edited or more taxes can be added, for example, some taxes used for services need to be
+manually added and configure, as depending on the city that you are offering the service, the rate
+may differ, and **taxes attached to services will not be computed by Avatax**, only good taxes are
+computed.
+
+When adding a tax used for a service that is included in the final price - when the tax is not added
+or subtracted on top of the product price - then you need to configure the :guilabel:`Tax
+Computation` to :guilabel:`Percentage of Price Tax Included`, and on the :guilabel:`Advanced
+Options` tab check the :guilabel:`Included in Price` option.
+
+.. image:: brazil/tax-configuration.png
```

Comment:
@samueljlieber also this image is wrong, I retook the screenshot.
Here is the one to be used here: https://drive.google.com/file/d/1w0JnmUjeY_iycauVkPyYUZ90bKKpehbK/view?usp=sharing

Sorry for that.